server/src/main/java/io/spine/server/procman/LifecycleRules.java

```diff
@@ -63,8 +63,8 @@ public LifecycleRules() {
     /**
      * Configures the repository to archive its entities when certain events/rejections occur.
      *
-     * <p>Subsequent calls to this method do not clear the previously added event classes (i.e. not
-     * override each other).
+     * <p>Subsequent calls to this method do not clear the previously added event classes,
+     * i.e, in order not to override each other.
```

Comment:
Why don't we have the period after `e` and before comma?

Also, it still does not read well. It was meant to say that we do not override events. Please ask @dmitrykuzmin for the meaning he had when writing this.

Reply:
I meant that in case we make two subsequent calls
```
lifecycle.archiveOn(Event1.class);
lifecycle.archiveOn(Event2.class);
```
the repository will archive when _either_ of `Event1` and `Event2` occurs and not only on `Event2`. I.e. subsequent calls to the method do not override the previous calls.

The same applies to the `deleteOn` method below.

Reply:
@dmitrykuzmin 
Thank you for clarifying! 
I will include more details in both entries then.